### PR TITLE
Docs/Add conversion code to the musicgen docs

### PR DESCRIPTION
**Pull request by yoinked-h**
# What does this PR do?

Adds conversion code to the musicgen docs (the conversion code is quite hidden and needs some changes to be run on other model dirs)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@sanchit-gandhi

### CODE REVIEW

**Comment by ArthurZucker** [comment]
Hey! Updated the people you pinged, let's minimize it 😉

File: docs/source/en/model_doc/musicgen.md [hunk 1]
```diff
@@ -46,6 +46,14 @@ This model was contributed by [sanchit-gandhi](https://huggingface.co/sanchit-ga
...
 [Hugging Face Hub](https://huggingface.co/models?sort=downloads&search=facebook%2Fmusicgen-).
 
+Note: Models exported via [this](https://github.com/facebookresearch/audiocraft/blob/main/docs/MUSICGEN.md#importing--exporting-models) 
+need to be converted to transformers before being run. This can be done with the script located at
+`src/transformers/models/musicgen/convert_musicgen_transformers.py`, like so:
+```
+python src/transformers/models/musicgen/convert_musicgen_transformers.py \
+    --checkpoint small --pytorch_dump_folder /output/path --safe_serialization --push_to_hub user/repo
+```
+
```
Comment (ArthurZucker) [inline docs/source/en/model_doc/musicgen.md:56]:
```suggestion
## Usage tips

- After downloading the original checkpoints from [here](https://github.com/facebookresearch/audiocraft/blob/main/docs/MUSICGEN.md#importing--exporting-models) , you can convert them using the **conversion script** available at
`src/transformers/models/musicgen/convert_musicgen_transformers.py` with the following command:

```bash
python src/transformers/models/musicgen/convert_musicgen_transformers.py \
    --checkpoint small --pytorch_dump_folder /output/path --safe_serialization 
```

```

Comment (yoinked-h) [reply docs/source/en/model_doc/musicgen.md:56]: seems good! (missing a triple backtick at the end, but added it on my fork)

**Review by ArthurZucker — APPROVED** [review]
Thanks 😉

**Pull request merged**